src/python/pants/report.py

```diff
@@ -0,0 +1,20 @@
+import uuid
```

Comment:
This file should live under src/python/pants/goal. Right now there is no BUILD target that covers it, which is why the CI is failing.

Reply:
But we can move that along with the other fixes, this PR is for discussion, we may not actually merge it as-is.

Reply:
Ah nice, I kept wondering about this. Thanks for clarification @benjyw  :)